examples/pruner/README.md

```diff
@@ -0,0 +1,29 @@
+# Pruning images using CronJob
+
+This example shows an image pruning happening in an automated fashion using the Kubernetes CronJobs that
+are available in OpenShift Origin from version 3.5.
+In this example, we will create a CronJob that will run image pruning every 15 minutes.
+
+## Requirements
+
+In order to execute the pruning commands successfully it is necessary to configure the
+authorization in a way that allows the `defaultq service account to perform the pruning
+against entire cluster (assuming you create the CronJob in the `default` project):
+
+1. `oc adm policy add-cluster-role-to-user sudoer admin-user --config=PATH/admin.kubeconfig`
+    This step is optional and it allows the "admin-user" to "sudo" into cluster
+    administrator role when needed.
+2. `oc adm policy add-cluster-role-to-user system:image-pruner system:serviceaccount:default:default --as=system:admin`
+    This command will grant the "image-pruner role to service account in the
+    `default` namespace. That will allow the service account list all images in the
+    cluster and perform the image pruning. Note that the `--as` is optional and you don't
+    have to use that option if you are already `system:admin` and you use `--config`.
+
+## Instaling the CronJob
+
+3. `oc create -f examples/pruner/job.yaml -n default --as=system:admin`
+    This command creates the CronJob resource that run the pruning job every 15 minutes.
+
+Make sure, that you check the `oc adm prune --help` command and optionally tweak the
+CronJob arguments by specifying how much tag revisions you want to preserve on a single tag
+or other options that might suit your environment.
```

Comment:
I'd add a note about removing old jobs, so that users are not surprised. Or maybe add a label to all of those jobs and a one-liner to remove them using that label.